### ISSUE TITLE
How are Semantic Kitti sequences 11-21 tested?

### ISSUE THREAD

**Issue by xiaohe3625** [yanx27/2DPASS#44]
Thank you for your great work. I achieved 68.587% mIou (num_vote=1) on validation set 08 using a pre-trained model, and then I want to test sequence 11-21, how do I do this?

**Comment by yanx27** [comment]
You can use `--submit_to_server` during the testing.

**Comment by brahami14** [comment]
Hello! 
thank you for answer, in this case we can't calculate the accuracy beacause we don't have the ground truth , isn'it ?

**Comment by yanx27** [comment]
@brahami14 Yes, you need to submit the prediction results on the official server.
